src/open_inwoner/openklant/views/contactform.py

```diff
@@ -36,6 +35,15 @@ def page_title(self):
     def get_success_url(self):
         return self.request.path
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs.update(
```

Comment:
I thought you were not a fun of `update` and you would assign it directly!

Reply:
Yea I copied this for multiple fields but forgot to change it when it was only one.